markdown/dev/howtos/ways-to-contribute/technical-writing/patterns/en.md

```diff
@@ -2,6 +2,13 @@
 title: Writing for freesewing.org
 ---
 
-You could write documentation for freesewing.org, our makers website.
+A good way of contributing to FreeSewing is writting documentation! 
 
-You would need good writing skills and a familiarity with sewing.
+We don't have specific guidelines on how to write documentation. You can write as you feel fit as long as it's relevant and follows [our Code of Conduct](/guides/code-of-conduct/) and [our community standards](https://freesewing.org/docs/various/community-standards/).
```

Comment:
I don't think this is the place to mention this. 
None of the other entries under *ways to contribute* specify these requirements because they are implied. It feels like a distraction to me to add them here.